### ISSUE TITLE
How can I create a XX_wordlist.combined.gz file?

### ISSUE THREAD

**Issue by Cyberman-tM** [AnySoftKeyboard/AnySoftKeyboard#2112]
The listed resources don't have it, and I haven't been able to figure out how to create one.
If I understand the instructions right, it's not necessarily required to have, as the build process can generate the wordlist out of raw text files.
But it probably would be cleaner to have that file?

Or am I maybe completely missing something?

**Comment by martholomew** [comment]
You can find a lot of them [here](https://github.com/LineageOS/android_packages_inputmethods_LatinIME/tree/lineage-16.0/dictionaries). If you want to generate one for a language that isn't there or otherwise, I have a script and instructions [here](https://github.com/martholomew/make-dic).

**Comment by Cyberman-tM** [comment]
Hm. Thank you, but it seems I'm still missing something - maybe it's too early (just got up an hour ago).

freq.sh doesn't like my file - probably because I'm not sure what you mean by "Edit freq.sh's first line to contain the unicode characters that you need (unicode-table.com is good for this)."

My file contains a lot of words separated by blanks and newlines. Apparently that isn't good enough for the script?

**Comment by martholomew** [comment]
What language is it in?

**Comment by Cyberman-tM** [comment]
Klingon :-)

No special characters apart from the apostrophe.

**Comment by martholomew** [comment]
`grep -Po "[A-Za-z\']+"`
That should do it!

**Comment by Cyberman-tM** [comment]
Ah, now I understand - the regex is there to capture the words. I thought it was something different.
It's running right now - as you wrote, it takes a while.
Thanks!

**Comment by martholomew** [comment]
Yeah, that's more important for language with different scripts, lol. No prob!

**Comment by Cyberman-tM** [comment]
Hm. OK, I get a result now - it looks different from what is in the main package. Similar, but not exactly the same.

I think I'm spending too much time on that by now - I should get back to the actual layout :-)

Thanks, though. I'll come back here sooner or later :-D
